src/PKSim.BatchTool/Services/BatchLogger.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Events;
+using OSPSuite.Core.Services;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Events;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace PKSim.BatchTool.Services
+{
+   public class BatchLogger : ILogger
+   {
```

Comment:
I Am going to rename that to PresentationLogger. That's the one logging in the UI